lib/keys.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Creates keys for `private` properties on exposed objects to minimize interactions with other codebases.
+ * The key will be a Symbol if the host supports it; otherwise a prefixed string.
+ */
+var create = 
+  typeof Symbol !== 'undefined' ? 
+  function(name) {
+    return Symbol(name);
+  } 
+  :
+  function(name) {
+    return '_zone$' + name;
+  };
+
+var commonKeys = {
+  originalAddEventListener: create('originalAddEventListener'),
```

Comment:
1) WRT `exports.common`, I prefer having the common keys on that object rather than being exported directly, so as to differentiate the common/shared keys from any (more general/abstract) key related functions. What if we need a common key called `create` in the future?

2) WRT the 'original' prefix, I carried that over from the _ prop name in `patchEventTargetMethods`. It refers to the 'original' method implementation rather than the patch, so personally I like having that explicit in the name. I don't mind either way though so it's up to you.